standalone-metastore/src/main/sql/mssql/upgrade-2.3.0-to-3.0.0.mssql.sql

```diff
@@ -214,3 +214,24 @@ ALTER TABLE COMPLETED_TXN_COMPONENTS ADD CTC_WRITEID bigint;
  ALTER TABLE KEY_CONSTRAINTS ADD DEFAULT_VALUE VARCHAR(400);
 
 ALTER TABLE HIVE_LOCKS MODIFY ALTER COLUMN HL_TXNID bigint NOT NULL;
+
+CREATE TABLE REPL_TXN_MAP (
+  RTM_REPL_POLICY nvarchar(256) NOT NULL,
+  RTM_SRC_TXN_ID bigint NOT NULL,
+  RTM_TARGET_TXN_ID bigint NOT NULL
+);
```

Comment:
Instead of ALTER TABLE command, can we use 
PRIMARY KEY CLUSTERED
(
    RTM_REPL_POLICY,  RTM_SRC_TXN_ID ASC
)

Reply:
its done for other table in same way ..so followed the same convention